spec/test_runner_spec.rb

```diff
@@ -7,6 +7,22 @@
   let(:file_multi) { File.new('spec/data/sample_multi.rb') }
   let(:file_failed) { File.new('spec/data/sample_failed.rb') }
 
+  describe '#compile_file_content' do
+    let(:test) { %q{
+y = 25
+#...content...#
+expect(x).to eq 35 } }
+    let(:expected) { %q{
+x = y + 10
+
+y = 25
+x = y + 10
```

Comment:
Your test is wrong. 

The expected result should be

```ruby
y = 25
x = y + 10
expect(x).to eq 35
```

However, it is not working, either. I am going to fix this